markhor_bringup/launch/cameras/markhor_video3_cam.launch

```diff
@@ -0,0 +1,10 @@
+<launch>
+    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" ns="markhor/camera_03" output="screen">
+        <param name="video_device" value="/dev/video3" />
+        <param name="image_width" value="640" />
```

Comment:
Also, we could try to increase the resolution? https://shop.orbbec3d.com/Astra-Embedded-S. We can have 720p @ 30 FPS. It'll be interesting to see what is the outcome. Although that can be tested after the competition (good first issue maybe?)

Reply:
Added to Teams tasks!